redwood-yoga/build.gradle

```
@@ -0,0 +1,18 @@
+import app.cash.redwood.buildsupport.KmpTargets
+
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+apply plugin: 'kotlinx-atomicfu'
+apply plugin: 'com.vanniktech.maven.publish'
+apply plugin: 'org.jetbrains.dokka' // Must be applied here for publish plugin.
```

Please use the new build plugin for this. Should be something like
```suggestion
apply plugin: 'app.cash.redwood.build'

redwoodBuild {
  publishing()
}
```